src/pages/about/Props1/props1.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './props1.scss'
+import { Link } from 'react-router-dom';
+
+interface IElement {
+    link: string;
+    title: string;
+    text: string;
+}
+export default function Props1 (props: IElement) {
```

Comment:
could you give an understandable name to this component?
and then you need to give a correct name for the interface (according to naming conventions)

Reply:
> could you give an understandable name to this component? and then you need to give a correct name for the interface (according to naming conventions)

I wrote something random because I tested working with props.